build.sbt

```diff
@@ -258,7 +260,7 @@ lazy val bazelProbeDriver =
   module(id = "bazel-probe-driver", path = "extensions/bazel/driver")
     .settings(
       name := "bazel-probe-driver",
-      libraryDependencies += "commons-codec" % "commons-codec" % "1.15"
+      libraryDependencies ++= Seq(Dependencies.commonsCodec, Dependencies.commonsIO)
```

Comment:
Added commons-io, which was available indirectly, as a transitive dependency of remote-robot (via logback-core). Once remote-robot removed its dependency on logback (https://github.com/JetBrains/intellij-ui-test-robot/issues/191), now commons-io wasn't available to bazel-probe-driver.